packages/x-handlebars/index.js

```
@@ -1,3 +1,40 @@
-module.exports = {
-	'x-teaser': require('./helpers/teaser')
+const { render } = require('@financial-times/x-engine');
+const extend = require('./concerns/extend');
+const resolvePeer = require('./concerns/resolve-peer');
+const resolveLocal = require('./concerns/resolve-local');
+
+// This is a regular function expression so that the template context may be shared as "this"
+const x = function ({ hash }) {
+	let moduleId;
+
+	if (hash.hasOwnProperty('package')) {
```

this should throw early if both `package` and `local` are missing